app/build.gradle

```diff
@@ -78,14 +81,17 @@ dependencies {
   // JSON serialization
   compile 'com.google.code.gson:gson:2.6.2'
   // Crash & usage monitoring
-  compile('com.crashlytics.sdk.android:crashlytics:2.5.5@aar') {
+  compile('com.crashlytics.sdk.android:crashlytics:2.6.5@aar') {
     transitive = true;
   }
   // Generate fake data
   compile 'com.github.blocoio:faker:1.2.3'
+  //EventBus
+  compile 'org.greenrobot:eventbus:3.0.0'
   // Testing
   testCompile 'junit:junit:4.12'
   testCompile 'org.mockito:mockito-core:2.0.31-beta'
+  androidTestCompile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.1'
```

Comment:
Yup, I think you can disable that constraint on Espresso, although the recommended way is to use an helper to scroll to the desired item.